CMakeLists.txt

```diff
@@ -203,10 +202,6 @@ endif()
 list(APPEND QUIC_COMMON_DEFINES VER_BUILD_ID=${QUIC_VER_BUILD_ID})
 list(APPEND QUIC_COMMON_DEFINES VER_SUFFIX=${QUIC_VER_SUFFIX})
 
-if(QUIC_RANDOM_ALLOC_FAIL)
-    list(APPEND QUIC_COMMON_DEFINES QUIC_RANDOM_ALLOC_FAIL=1 QUIC_DISABLE_MEM_POOL=1)
```

Comment:
Looks like `QUIC_DISABLE_MEM_POOL` will be a good "next step" to enable at run time for spin quic.